README.md

```diff
@@ -87,40 +87,28 @@ To launch **Eppie Console** application, run the following command:
 dotnet run --project ./src/Eppie.CLI/Eppie.CLI/Eppie.CLI.csproj
 ```
 
-To use **Gmail** or **Microsoft Outlook** OAuth2 authorization, you need to [register the application](docs/Register%20the%20application.md) and then to pass the **ClientId** and **ClientSecret** arguments with the command:
+If you compile your own binaries, you will need to authorize the application in **Google Developer Console** and **Microsoft Azure Portal** in order to be able to connect **Gmail** and **Microsoft Outlook** mailboxes. Basically, Google and Microsoft want to know a little bit about the app before they allow it to access their users' data. [Here](docs/Register%20the%20application.md) is a little tutorial. When you get the **Client ID** and **Client Secret** (only Gmail) pass them as arguments with launch command, like this:
 
 ```console
 dotnet run --project ./src/Eppie.CLI/Eppie.CLI/Eppie.CLI.csproj -- --Authorization:Gmail:ClientId="<gmail-client-id>" --Authorization:Gmail:ClientSecret="<gmail-client-secret>" --Authorization:Outlook:ClientId="<outlook-client-id>"
 ```
 
 ### If downloaded the binaries
 
-Run this while in the project folder
+Previously users were required to authorize Eppie with Google and Microsoft and pass the credentials they'd created to the app. This is no longer the case if you download the latest executables. Just run this while in the project folder:
```

Comment:
I think _"Previously users were required... the latest executables."_ text should be removed. And add notes about running the **"chmod +x eppie-console"** command before running the eppie-console on linux and macos.

Reply:
done